package.json

```diff
@@ -28,6 +28,7 @@
     "commander": "2.3.0",
     "component-classes": "1.2.1",
     "component-closest": "0.1.4",
+    "component-file-picker": "0.2.1",
```

Comment:
https://github.com/Automattic/wp-calypso/blob/master/docs/shrinkwrap.md#modifying-dependencies

And we're currently using node 4.3.0 with npm 2.14.12